source/includes/usage-examples/read-write-pref.rb

```diff
@@ -0,0 +1,77 @@
+require 'bundler/inline'
+
+gemfile do
+  source 'https://rubygems.org'
+  gem 'mongo'
+end
+
+uri = '<connection string>'
+
+Mongo::Client.new(uri) do |client|
+    # Access the database and collection
+
+    # start-write-concern
+    client = Mongo::Client.new(['IP_ADDRESS_001:27017'], database: 'myDB')
+    myDB = client.database
+    myCollection = myDB[:myCollection]
+
+    myCollection.insert_one(
+      { name: "anotherDocumentName" },
+      write: {
+        w: 2,
+        wtimeout: 5000
+      }
+    )
+    # end-write-concern
+
+    # start-write-concern-2
+    myDoc = { name: "New Document" }
+    new_write_concern = Mongo::WriteConcern.get(myDB.write_concern)
+    myDB[:myCollection].with(write: new_write_concern).insert_one(myDoc)
+    # end-write-concern-2
+
+    # start-read-concern
+    pipeline = [
+    { "$match" => { category: "KITCHENWARE" } },
+    { "$unset" => ["_id", "category"] }
+    ]
+    result = myCollection.aggregate(pipeline, read: { read_concern: { level: :available } })
```

Comment:
Spacing for readability
```suggestion
    pipeline = [
      { "$match" => { category: "KITCHENWARE" } },
      { "$unset" => ["_id", "category"] }
    ]
    result = myCollection.aggregate(pipeline, 
      read: { read_concern: { level: :available } })
```